pom.xml

```diff
@@ -98,9 +98,33 @@
               <additionalClasspathElement>${maven.jdbc.driver.jar}</additionalClasspathElement>
             </additionalClasspathElements>
           </configuration>
-        </plugin> 
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
```

Comment:
My $0.02: this configuration should be centralized across the entire KIE Group. Individual projects would then simply opt-in by enabling it. This will make it much easier to propagate this change in the future, and make the enforcement rules consistent across the entire codebase.